src/components/ThemeToggleButton.js

```diff
@@ -24,7 +25,10 @@ const ThemeToggleButton = () => {
         <Button
           aria-label="Toggle theme"
           variant="ghost"
-          onClick={toggleColorMode}
+          onClick={() => {
+            toggleColorMode();
+            reportEvent("header", "click", "color_mode");
```

Comment:
Maybe we can see which color mode is more popular by doing this.
```suggestion
            reportEvent("header", "click", useColorModeValue("useDarkMode", "useBrightMode"));
```